CONTRIBUTING.md

```diff
@@ -229,13 +229,13 @@ The documentation work is all done in [this](https://github.com/rackerlabs/docs-
 2. Once you have created a pull request, tag the doc team for review. See [Documentation team FAQ](https://github.com/rackerlabs/docs-rackspace/blob/master/doc/contributor-collateral/privatecloud-docteam-FAQ.rst) for clarification on who to contact.
 3. Await merge. Respond to any questions/comments/concerns.
 
-Any further questions about our github processes, contributor guidelines, or FAQ about the docs and team, refer to: [Contributor collatoral](https://github.com/rackerlabs/docs-rackspace/tree/master/doc/contributor-collateral) 
+Any further questions about our github processes, contributor guidelines, or FAQ about the docs and team, refer to: [Contributor collatoral](https://github.com/rackerlabs/docs-rackspace/tree/master/doc/contributor-collateral)
 
 ## Commits, release notes and pull requests
 ### Fixing a bug
 
-1. After the bug is triaged and prioritised, make the required changes. New features, breaking changes and other patches of note must include a release note generated using the `reno tool`. Please see `Release Notes` for more information.
-2. Push changes directly to a branch on ```rpc-openstack``` in the ```rcbops``` github namespace (rather than a developers own fork). A pull request is then made from this branch to the ```master``` branch.
+1. After the bug is triaged and prioritised, make the required changes. New features, breaking changes and other patches of note must include a release note generated using the `reno` tool. Please see `Release Notes` for more information.
+2. Push changes directly to a branch on ```rpc-openstack``` in the ```rcbops``` github namespace (rather than a developer's own fork). A pull request is then made from this branch to the ```master``` branch.
```

Comment:
I know this change only corrects grammar however after reading I find myself asking why are we stating how a developer must work in-order to contribute to this project? As a developer I will never do this because I will always work on items in my fork within a bug or feature branch and create pull requests from their when I believe they're ready. If we're wanting to allow for outside contributions it seems like a very bad idea to force folks into a single type of workflow. Would it not be better to require folks to squash their commits within their private forks and submit a PR from there?